### PR TITLE
New language dictionary

### PR DESCRIPTION
**Pull request by rprzedzik**

### CODE REVIEW

File: module/designer/features/designer.feature [hunk 1]
```diff
@@ -418,315 +418,315 @@ Feature: Designer module
...
   Scenario: Get templates types (order ASC)
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/templates/types?field=type&order=ASC"
+    When I send a GET request to "/api/v1/en/templates/types?field=type&order=ASC"
     Then the JSON should be valid according to the schema "module/grid/features/gridSchema.json"
 
-  Scenario: Get templates types (order DESC)
+  Scenario: Get templates types (order deSC)
```
Comment (wiewiurdp) [inline module/designer/features/designer.feature]: It this change really needed ?

File: module/designer/features/designer.feature [hunk 2]
```diff
@@ -418,315 +418,315 @@ Feature: Designer module
...
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/templates/types?field=type&order=DESC"
+    When I send a GET request to "/api/v1/en/templates/types?field=type&order=deSC"
```
Comment (wiewiurdp) [inline module/designer/features/designer.feature]: It this change really needed ?

File: module/product/features/product-grid.feature [hunk 3]
```diff
@@ -487,33 +487,33 @@ Feature: Product edit feature
...
-    When I send a GET request to "api/v1/EN/products?columns=@product_edit_numeric_attribute@&filter=@product_edit_numeric_attribute@%3E%3D1%3B@product_edit_numeric_attribute@%3C%3D3"
+    When I send a GET request to "api/v1/en/products?columns=@product_edit_numeric_attribute@&filter=@product_edit_numeric_attribute@%3E%3D1%3B@product_edit_numeric_attribute@%3C%3D3"
     Then the response status code should be 200
 
   Scenario: Request product order by index
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "api/v1/EN/products?columns=@product_edit_text_attribute_code@&,index&field=index&order=DESC"
+    When I send a GET request to "api/v1/en/products?columns=@product_edit_text_attribute_code@&,index&field=index&order=deSC"
```
Comment (wiewiurdp) [inline module/product/features/product-grid.feature]: It this change really needed ?

File: module/product/features/product-grid.feature [hunk 4]
```diff
@@ -487,33 +487,33 @@ Feature: Product edit feature
...
     Then the response status code should be 200
 
   Scenario: Request product order by attribute
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "api/v1/EN/products?columns=@product_edit_text_attribute_code@&,index&field=@product_edit_text_attribute_code@&order=DESC"
+    When I send a GET request to "api/v1/en/products?columns=@product_edit_text_attribute_code@&,index&field=@product_edit_text_attribute_code@&order=deSC"
```
Comment (wiewiurdp) [inline module/product/features/product-grid.feature]: It this change really needed ?

File: module/product/features/product.feature [hunk 5]
```diff
@@ -313,29 +313,29 @@ Feature: Product module
...
-    When I send a GET request to "/api/v1/EN/products/@product@/collections"
+    When I send a GET request to "/api/v1/en/products/@product@/collections"
     Then the response status code should be 401
 
   Scenario: Get product collection element (order by code)
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/products/@product@/collections?field=code&order=DESC"
+    When I send a GET request to "/api/v1/en/products/@product@/collections?field=code&order=deSC"
```
Comment (wiewiurdp) [inline module/product/features/product.feature]: It this change really needed ?

File: module/product/features/product.feature [hunk 6]
```diff
@@ -344,7 +344,7 @@ Feature: Product module
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/products/@product@/collections?field=name&order=DESC"
+    When I send a GET request to "/api/v1/en/products/@product@/collections?field=name&order=deSC"
```
Comment (wiewiurdp) [inline module/product/features/product.feature]: It this change really needed ?

File: module/product/features/product.feature [hunk 7]
```diff
@@ -354,7 +354,7 @@ Feature: Product module
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/products/@product@/collections?field=description&order=DESC"
+    When I send a GET request to "/api/v1/en/products/@product@/collections?field=description&order=deSC"
```
Comment (wiewiurdp) [inline module/product/features/product.feature]: It this change really needed ?

File: module/product/features/product.feature [hunk 8]
```diff
@@ -363,7 +363,7 @@ Feature: Product module
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/products/@product@/collections?field=type_id&order=DESC"
+    When I send a GET request to "/api/v1/en/products/@product@/collections?field=type_id&order=deSC"
```
Comment (wiewiurdp) [inline module/product/features/product.feature]: It this change really needed ?

File: module/product/features/product.feature [hunk 9]
```diff
@@ -372,7 +372,7 @@ Feature: Product module
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/products/@product@/collections?field=elements_count&order=DESC"
+    When I send a GET request to "/api/v1/en/products/@product@/collections?field=elements_count&order=deSC"
```
Comment (wiewiurdp) [inline module/product/features/product.feature]: It this change really needed ?

File: module/product/features/product.feature [hunk 10]
```diff
@@ -409,83 +409,83 @@ Feature: Product module
...
+    When I send a GET request to "/api/v1/en/products?field=index&order=ASC"
     Then the JSON should be valid according to the schema "module/grid/features/gridSchema.json"
 
-  Scenario: Get products (order DESC)
+  Scenario: Get products (order deSC)
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/products?field=index&order=DESC"
+    When I send a GET request to "/api/v1/en/products?field=index&order=deSC"
```
Comment (wiewiurdp) [inline module/product/features/product.feature]: It this change really needed ?

File: module/workflow/features/workflow_status.feature [hunk 11]
```diff
@@ -26,37 +26,37 @@ Feature: Workflow
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a POST request to "/api/v1/EN/status" with body:
+    When I send a POST request to "/api/v1/en/status" with body:
       """
       {
         "color": "#ff0000",
-        "code": "DESTINATION @@random_md5@@",
+        "code": "deSTINATION @@random_md5@@",
```
Comment (wiewiurdp) [inline module/workflow/features/workflow_status.feature]: It this change really needed ?

File: module/workflow/features/workflow_status.feature [hunk 12]
```diff
@@ -119,8 +119,8 @@ Feature: Workflow
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    Given remember param "duplicated_status_code" with value "DESTINATION_1_@@random_code@@"
-    When I send a POST request to "/api/v1/EN/status" with body:
+    Given remember param "duplicated_status_code" with value "deSTINATION_1_@@random_code@@"
```
Comment (wiewiurdp) [inline module/workflow/features/workflow_status.feature]: It this change really needed ?

File: module/workflow/features/workflow_status.feature [hunk 13]
```diff
@@ -146,25 +146,25 @@ Feature: Workflow
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    Given remember param "duplicated_status_code" with value "DESTINATION_1_@@random_code@@"
-    When I send a PUT request to "/api/v1/EN/status/@workflow_first_status@" with body:
+    Given remember param "duplicated_status_code" with value "deSTINATION_1_@@random_code@@"
```
Comment (wiewiurdp) [inline module/workflow/features/workflow_status.feature]: It this change really needed ?

File: module/workflow/features/workflow_status.feature [hunk 14]
```diff
@@ -188,138 +188,138 @@ Feature: Workflow
...
   Scenario: Get statuses (order ASC)
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/status?field=name&order=ASC"
+    When I send a GET request to "/api/v1/en/status?field=name&order=ASC"
     Then the JSON should be valid according to the schema "module/grid/features/gridSchema.json"
 
-  Scenario: Get statuses (order DESC)
+  Scenario: Get statuses (order deSC)
```
Comment (wiewiurdp) [inline module/workflow/features/workflow_status.feature]: It this change really needed ?

File: module/workflow/features/workflow_status.feature [hunk 15]
```diff
@@ -188,138 +188,138 @@ Feature: Workflow
...
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/status?field=name&order=DESC"
+    When I send a GET request to "/api/v1/en/status?field=name&order=deSC"
```
Comment (wiewiurdp) [inline module/workflow/features/workflow_status.feature]: It this change really needed ?

File: module/workflow/features/workflow_transitions.feature [hunk 16]
```diff
@@ -41,18 +41,18 @@ Feature: Workflow
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a POST request to "/api/v1/EN/status" with body:
+    When I send a POST request to "/api/v1/en/status" with body:
       """
       {
         "color": "#ff0000",
-        "code": "DESTINATION @@random_md5@@",
+        "code": "deSTINATION @@random_md5@@",
```
Comment (wiewiurdp) [inline module/workflow/features/workflow_transitions.feature]: It this change really needed ?

File: module/attribute/features/attribute-grid.feature [hunk 17]
```diff
@@ -6,61 +6,61 @@ Feature: Attribute grid
...
     Then the response status code should be 200
 
   Scenario: Get attributes (order ASC)
-    And I send a "GET" request to "/api/v1/EN/attributes?field=label&order=ASC"
+    And I send a "GET" request to "/api/v1/en/attributes?field=label&order=ASC"
     Then the response status code should be 200
 
-  Scenario: Get attributes (order DESC)
-    And I send a "GET" request to "/api/v1/EN/attributes?field=label&order=DESC"
+  Scenario: Get attributes (order deSC)
```
Comment (wiewiurdp) [inline module/attribute/features/attribute-grid.feature]: It this change really needed ?

File: module/attribute/features/attribute-grid.feature [hunk 18]
```diff
@@ -6,61 +6,61 @@ Feature: Attribute grid
...
+    And I send a "GET" request to "/api/v1/en/attributes?field=label&order=deSC"
```
Comment (wiewiurdp) [inline module/attribute/features/attribute-grid.feature]: It this change really needed ?

File: module/category/features/category-tree.feature [hunk 19]
```diff
@@ -429,108 +429,108 @@ Feature: Category tree module
...
   Scenario: Get category trees (order ASC)
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/trees?field=name&order=ASC"
+    When I send a GET request to "/api/v1/en/trees?field=name&order=ASC"
     Then the JSON should be valid according to the schema "module/grid/features/gridSchema.json"
 
-  Scenario: Get category trees (order DESC)
+  Scenario: Get category trees (order deSC)
```
Comment (wiewiurdp) [inline module/category/features/category-tree.feature]: It this change really needed ?

File: module/category/features/category-tree.feature [hunk 20]
```diff
@@ -429,108 +429,108 @@ Feature: Category tree module
...
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/trees?field=name&order=DESC"
+    When I send a GET request to "/api/v1/en/trees?field=name&order=deSC"
```
Comment (wiewiurdp) [inline module/category/features/category-tree.feature]: It this change really needed ?

File: module/channel/features/channel.feature [hunk 21]
```diff
@@ -177,73 +177,73 @@ Feature: channel module
...
   Scenario: Get channels (order ASC)
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/channels?field=name&order=ASC"
+    When I send a GET request to "/api/v1/en/channels?field=name&order=ASC"
     Then the JSON should be valid according to the schema "module/grid/features/gridSchema.json"
 
-  Scenario: Get channels (order DESC)
+  Scenario: Get channels (order deSC)
```
Comment (wiewiurdp) [inline module/channel/features/channel.feature]: It this change really needed ?

File: module/channel/features/channel.feature [hunk 22]
```diff
@@ -177,73 +177,73 @@ Feature: channel module
...
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/channels?field=name&order=DESC"
+    When I send a GET request to "/api/v1/en/channels?field=name&order=deSC"
```
Comment (wiewiurdp) [inline module/channel/features/channel.feature]: It this change really needed ?

File: module/core/features/core.feature [hunk 23]
```diff
@@ -4,110 +4,110 @@ Feature: Core module
...
   Scenario: Get languages (order ASC)
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/languages?field=name&order=ASC"
+    When I send a GET request to "/api/v1/en/languages?field=name&order=ASC"
     Then the JSON should be valid according to the schema "module/grid/features/gridSchema.json"
 
-  Scenario: Get languages (order DESC)
+  Scenario: Get languages (order deSC)
```
Comment (wiewiurdp) [inline module/core/features/core.feature]: It this change really needed ?

File: module/core/features/core.feature [hunk 24]
```diff
@@ -4,110 +4,110 @@ Feature: Core module
...
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/languages?field=name&order=DESC"
+    When I send a GET request to "/api/v1/en/languages?field=name&order=deSC"
```
Comment (wiewiurdp) [inline module/core/features/core.feature]: It this change really needed ?

File: module/core/features/core.feature [hunk 25]
```diff
@@ -161,60 +161,57 @@ Feature: Core module
...
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/language/autocomplete?field=name&order=ASC"
+    When I send a GET request to "/api/v1/en/language/autocomplete?field=name&order=ASC"
     Then the response status code should be 200
     And the JSON should be valid according to the schema "module/core/features/language.json"
 
-  Scenario: Get language autocomplete (order DESC)
+  Scenario: Get language autocomplete (order deSC)
```
Comment (wiewiurdp) [inline module/core/features/core.feature]: It this change really needed ?

File: module/core/features/core.feature [hunk 26]
```diff
@@ -161,60 +161,57 @@ Feature: Core module
...
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/language/autocomplete?field=name&order=DESC"
+    When I send a GET request to "/api/v1/en/language/autocomplete?field=name&order=deSC"
```
Comment (wiewiurdp) [inline module/core/features/core.feature]: It this change really needed ?

File: module/core/features/unit.feature [hunk 27]
```diff
@@ -108,94 +108,94 @@ Feature: Core module - unit
...
   Scenario: Get units (order ASC)
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/units?field=name&order=ASC"
+    When I send a GET request to "/api/v1/en/units?field=name&order=ASC"
     Then the JSON should be valid according to the schema "module/grid/features/gridSchema.json"
 
-  Scenario: Get units (order DESC)
+  Scenario: Get units (order deSC)
```
Comment (wiewiurdp) [inline module/core/features/unit.feature]: It this change really needed ?

File: module/core/features/unit.feature [hunk 28]
```diff
@@ -108,94 +108,94 @@ Feature: Core module - unit
...
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/units?field=name&order=DESC"
+    When I send a GET request to "/api/v1/en/units?field=name&order=deSC"
```
Comment (wiewiurdp) [inline module/core/features/unit.feature]: It this change really needed ?

File: module/designer/features/designer.feature [hunk 29]
```diff
@@ -418,315 +418,315 @@ Feature: Designer module
...
   Scenario: Get templates (order ASC)
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/templates?field=name&order=ASC"
+    When I send a GET request to "/api/v1/en/templates?field=name&order=ASC"
     Then the JSON should be valid according to the schema "module/grid/features/gridSchema.json"
 
-  Scenario: Get templates (order DESC)
+  Scenario: Get templates (order deSC)
```
Comment (wiewiurdp) [inline module/designer/features/designer.feature]: It this change really needed ?

File: module/designer/features/designer.feature [hunk 30]
```diff
@@ -418,315 +418,315 @@ Feature: Designer module
...
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/templates?field=name&order=DESC"
+    When I send a GET request to "/api/v1/en/templates?field=name&order=deSC"
```
Comment (wiewiurdp) [inline module/designer/features/designer.feature]: It this change really needed ?

File: module/designer/features/designer.feature [hunk 31]
```diff
@@ -418,315 +418,315 @@ Feature: Designer module
...
   Scenario: Get templates groups (order ASC)
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/templates/groups?field=name&order=ASC"
+    When I send a GET request to "/api/v1/en/templates/groups?field=name&order=ASC"
     Then the JSON should be valid according to the schema "module/grid/features/gridSchema.json"
 
-  Scenario: Get templates groups (order DESC)
+  Scenario: Get templates groups (order deSC)
```
Comment (wiewiurdp) [inline module/designer/features/designer.feature]: It this change really needed ?

File: module/designer/features/designer.feature [hunk 32]
```diff
@@ -418,315 +418,315 @@ Feature: Designer module
...
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/templates/groups?field=name&order=DESC"
+    When I send a GET request to "/api/v1/en/templates/groups?field=name&order=deSC"
```
Comment (wiewiurdp) [inline module/designer/features/designer.feature]: It this change really needed ?

File: module/product/features/product.feature [hunk 33]
```diff
@@ -409,83 +409,83 @@ Feature: Product module
...
   Scenario: Get products (order ASC)
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a GET request to "/api/v1/EN/products?field=index&order=ASC"
+    When I send a GET request to "/api/v1/en/products?field=index&order=ASC"
     Then the JSON should be valid according to the schema "module/grid/features/gridSchema.json"
 
-  Scenario: Get products (order DESC)
+  Scenario: Get products (order deSC)
```
Comment (wiewiurdp) [inline module/product/features/product.feature]: It this change really needed ?

File: module/designer/features/designer.feature (hunk 2 above)
Comment (rprzedzik) [reply module/designer/features/designer.feature]: Yes

File: module/product/features/product-grid.feature (hunk 3 above)
Comment (rprzedzik) [reply module/product/features/product-grid.feature]: Yes

File: module/product/features/product-grid.feature [hunk 34]
```diff
@@ -105,10 +105,10 @@ Feature: Product edit feature
     Given I am Authenticated as "test@ergonode.com"
     And I add "Content-Type" header equal to "application/json"
     And I add "Accept" header equal to "application/json"
-    When I send a POST request to "/api/v1/EN/attributes" with body:
+    When I send a POST request to "/api/v1/en/attributes" with body:
       """
       {
-        "code": "LONG_CODE_ATTRIBUTE_1234567890_1234567890_1234567890_1234567890_1234567890_1234567890_@@random_code@@",
+        "code": "LONG_COde_ATTRIBUTE_1234567890_1234567890_1234567890_1234567890_1234567890_1234567890_@@random_code@@",
```
Comment (wfajczyk) [inline module/product/features/product-grid.feature:111]: `LONG_COde_ATTRIBUTE` to change but it's not a bug

**Pull request merged**